pendulum/tz/timezone.py

```diff
@@ -24,10 +24,10 @@ class Timezone(tzinfo):
     POST_TRANSITION = 'post'
     TRANSITION_ERROR = 'error'
 
-    def __init__(self, name, transitions,
-                 tzinfos,
-                 default_tzinfo_index,
-                 utc_transition_times):
+    def __init__(self, name=None, transitions=(),
+                 tzinfos=(),
+                 default_tzinfo_index=0,
+                 utc_transition_times=[]):
```

Comment:
Beyond this I think it'd be a good idea to implement `__copy__` and `__deepcopy__` which are [referenced in the Python docs for the `copy` module](https://docs.python.org/3/library/copy.html#copy.deepcopy) rather than adding default parameters. TBH I've never used these methods myself so some experimentation might be required.